multimedia/vapoursynth/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vapoursynth vapoursynth 52 R
+
+categories              multimedia python
+platforms               darwin
+license                 LGPL-2.1+
+
+description             A video processing framework with simplicity in mind
+
+long_description        VapourSynth is an application for video manipulation. \
+                        Or a plugin. Or a library. It’s hard to tell because \
+                        it has a core library written in C++ and a Python \
+                        module to allow video scripts to be created. The \
+                        software has been heavily inspired by Avisynth.
+
+checksums               rmd160  6af196e73295a76d8ef8e19506b7166beb722874 \
+                        sha256  a3623acdb68a131917a0c48b6d3c7061be1c585622699ae4b4289f4840ae7a96 \
+                        size    619067
+
+maintainers             {gmail.com:herby.gillot @herbygillot} \
+                        openmaintainer
+
+set python_version      3.8
+
+use_autoreconf          yes
+autoreconf.cmd          ./autogen.sh
+autoreconf.args
+
+depends_build-append    port:autoconf \
+                        port:automake \
+                        port:libtool \
+                        port:pkgconfig \
+                        port:py38-cython
```

Comment:
You could use a variable here too instead of repeating the python version. Unfortunately some occasions need the dot in the version number and some don't so we need two variables. The python portgroup uses the naming convention that "python.branch" is the number with the dot and "python.version" is the number without the dot. While that naming convention is unfortunate, it's what we have, so you might as well repeat it in this port.

```
set python_branch       3.8
set python_version      [string map {. {}} ${python_branch}]
```
```
                        port:py${python_version}-cython
```
```
                        port:python${python_version} \
```
```
configure.args          --with-cython=${prefix}/bin/cython-${python_branch}
```
```
configure.python        ${prefix}/bin/python${python_branch}
```

Reply:
Makes sense, applied.